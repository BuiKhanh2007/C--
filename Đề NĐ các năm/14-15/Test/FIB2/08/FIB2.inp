50
15 674
5 1
4 3
3 1
16 1180
31 1606825
19 5238
29 195774
21 2908
0 1
2 2
24 40032
25 85106
3 1
17 1921
28 268400
32 151587
18 113
31 1918205
23 42764
26 38583
23 12876
23 38833
28 506753
6 5
20 8476
9 36
7 10
18 2880
17 1507
27 82169
18 1526
3 3
23 20004
24 49327
27 52428
21 12056
26 52688
18 121
29 513379
13 271
12 228
30 154648
20 7164
3 1
16 196
30 721678
1 1
19 225
19 5095
