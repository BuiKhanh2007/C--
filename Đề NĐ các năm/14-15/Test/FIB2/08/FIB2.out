258
0
1
0
451
613753
2001
74779
1111
1
1
15291
32508
0
734
102520
57901
43
732689
16335
14738
4918
14833
193563
2
3238
14
4
1100
576
31386
583
1
7641
18841
20026
4605
20125
46
196093
104
87
59070
2737
0
75
275657
0
86
1946
